playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -43,11 +43,14 @@
   - name: Evaluate groups - Fail if no etcd hosts group is defined
     fail:
       msg: >
-        No etcd hosts defined. Running an all-in-one master is deprecated and
-        will no longer be supported in a future upgrade.
+        No etcd hosts defined. Running embedded etcd is no longer be supported.
```

Comment:
Running embedded etcd is no longer supported. **

Reply:
better?

Reply:
Nice